packages/lesswrong/themes/siteThemes/eaTheme.ts

```diff
@@ -65,6 +65,7 @@ export const eaForumTheme: SiteThemeSpecification = {
     individual: '#BF577D',
     icon: {
       navigationSidebarIcon: shadePalette.greyAlpha(0.5),
+      sprout: '#76a47e'
```

Comment:
thanks! Changing it to a slightly more clear green so it doesn't look grey at a glance. Still not as prominent as in Figma though 
```suggestion
      sprout: '#5EB25C'
```